nevergrad/benchmark/test_xpbase.py

```diff
@@ -48,21 +48,25 @@ def test_run_packed_artificial_function() -> None:
     )
     xp = xpbase.Experiment(func, optimizer="OnePlusOne", budget=24, num_workers=2, batch_mode=True, seed=14)
     summary = xp.run()
-    np.testing.assert_almost_equal(summary["loss"], -9676.5, decimal=1)  # makes sure seeding works!
+    np.testing.assert_almost_equal(
+        summary["loss"], -9784.829729792353, decimal=1
+    )  # makes sure seeding works!
 
 
 def test_noisy_artificial_function_loss() -> None:
     func = ArtificialFunction(name="sphere", block_dimension=5, noise_level=0.3)
     seed = np.random.randint(99999)
     xp = xpbase.Experiment(func, optimizer="OnePlusOne", budget=5, seed=seed)
+    # Because copy() can have different random initialization for the parameters
+    # The function should be copied early.
+    np.random.seed(seed)
+    pfunc = func.copy()
```

Comment:
can you elaborate? I don't get it, I would have expected to work the same from anywhere.